plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/core-profiler.spec.js

```diff
@@ -347,27 +349,45 @@ test.describe(
 					} )
 				).toBeVisible();
 				// confirm that the optional plugins are present
-				await expect(
-					page.locator( '.plugin-title', {
-						hasText: 'Pinterest for WooCommerce',
-					} )
-				).toBeVisible();
-				await expect(
-					page.locator( '.plugin-title', {
-						hasText: /Google for WooCommerce|Google Listings & Ads/,
-					} )
-				).toBeVisible();
+				try {
```

Comment:
could extract the try-catch and parameterise it to reduce the copy pasta 😬 just a comment, it's fine as is too

Reply:
Appreciate the suggestion, @rjchow! I think it's fine as-is for now, I'd prefer to make such changes in a future refactor 🤞